### ISSUE TITLE
low miou when using pspnet and voc dataset

### ISSUE THREAD

**Issue by littlebelly** [meetshah1995/pytorch-semseg#59]
Hi, thanks for your excellent work!

But I got low miou when I use pspnet and load pretrained model on pascal voc 2012 val dataset.

There are my results:
Overall Acc: 	 0.651600544826
Mean IoU : 	 0.0329240353333
Mean Acc : 	 0.0462086706718
FreqW Acc : 	 0.495069499643

and when I validate the same data on original caffe version, I got miou about 91%

Could you help me to solve this problem? Thanks alot!

**Comment by meetshah1995** [comment]
Fixed a bug in pyramid poolingeven by e4a02b40121db76feee25b6c56172a66650907aa

There is still a bug in inference due to different outputs in nn.Batchnorm even in eval mode with same input.
